mylyn.builds/org.eclipse.mylyn.hudson.core/META-INF/MANIFEST.MF

```diff
@@ -4,14 +4,21 @@ Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.eclipse.mylyn.hudson.core;singleton:=true
 Bundle-Version: 3.26.0.qualifier
 Bundle-Vendor: %Bundle-Vendor
-Require-Bundle: jakarta.xml.bind;bundle-version="2.3.3",
- org.eclipse.core.runtime;bundle-version="0.0.0",
+Require-Bundle: org.eclipse.core.runtime;bundle-version="0.0.0",
  org.eclipse.emf.ecore;bundle-version="0.0.0",
  org.eclipse.mylyn.builds.core;bundle-version="3.26.0",
  org.eclipse.mylyn.commons.core;bundle-version="3.26.0",
  org.eclipse.mylyn.commons.repositories.core;bundle-version="3.26.0",
  org.eclipse.mylyn.commons.repositories.http.core;bundle-version="3.26.0",
- org.eclipse.mylyn.hudson.core;bundle-version="3.26.0"
+ org.eclipse.mylyn.hudson.core;bundle-version="3.26.0",
```

Comment:
Can't you use import package instead of require bundle? Also, are all of these required?